users/imchipwood/imchipwood.c

```diff
@@ -0,0 +1,126 @@
+#include "imchipwood.h"
+
+static td_state_t td_state;
```

Comment:
Heads up, if you have more than one dance using this, .... don't. 

You want one struct for each dance.  So, 3 from the looks of it.  

```suggestion
static td_state_t td_state[3];
```

Reply:
Got it - thanks!